in-development/mission_3774_sap_task_center/README.md

```diff
@@ -0,0 +1,82 @@
+# Discovery Center mission - Establish a Central Inbox with SAP Task Center
+
+## Overview
+
+This sample shows how to setup your SAP BTP account for the Discovery Center Mission - [Establish a Central Inbox with SAP Task Center](https://discovery-center.cloud.sap/index.html#/missiondetail/3774/)
+
+## Content of setup
+
+The setup comprises the following resources:
+
+- Creation of the SAP BTP subaccount
+- Entitlements of services
+- Subscriptions to applications
+- Creation of service instance
+- Role collection assignments to users
+
+## Deploying the resources
+
+Make sure that you are familiar with SAP BTP and know both the [Get Started with btp-terraform-samples](https://github.com/SAP-samples/btp-terraform-samples/blob/main/GET_STARTED.md) and the [Get Started with the Terraform Provider for BTP](https://developers.sap.com/tutorials/btp-terraform-get-started.html)
+
+To deploy the resources you must:
+
+1. Set the environment variables BTP_USERNAME and BTP_PASSWORD to pass credentials to the BTP provider to authenticate and interact with your BTP environments. 
+
+   ```bash
+   export BTP_USERNAME=<your_username>
+   export BTP_PASSWORD=<your_password>
+   ```
```

Comment:
The information about the CF authentication is missing see https://registry.terraform.io/providers/SAP/cloudfoundry/latest/docs#optional